enarx-keep-sev/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "enarx-keep-sev"
+version = "0.1.0"
+authors = ["Harald Hoyer <harald@redhat.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+failure = "0.1.7"
```

Comment:
hmm..  I thought `failure` is dead? How about `anyhow`?

Reply:
:scream: all my research led me to `failure`! Indeed, the repository doesn't look very active.

It's a shame I kind of like the crate overall though. Let me look and see what's out there.

Reply:
Yeah, `failure` was a try from withoutboats to tackle error handling, but I think he moved on to other things.

Meanwhile I created [chainerror](https://haraldh.github.io/chainerror/), which gives a nice error traceback (different from the panic backtrace). Also I added https://doc.rust-lang.org/std/error/trait.Error.html#method.chain but it's nightly only.

From my experience with error handling, if you are writing a `lib` crate, use something like https://github.com/tailhook/quick-error, and implement s.th. like `Chainerror` [manually](https://github.com/varlink/rust/blob/master/varlink/src/error.rs#L117-L140)

If you are creating a `bin` crate, you might use `anyhow`, but I prefer manual implementation.

Reply:
See also this: https://www.youtube.com/watch?v=qWY2sUpt-Ek

I think we should have a discussion about error handling. My knowledge here is very poor. But I also know we need good error handling across a variety of contexts (std, no_std, no_std w/ nostdlib).

Reply:
In the meantime, failure is dead so lets remove it.

Reply:
> I think we should have a discussion about error handling. My knowledge here is very poor. But I also know we need good error handling across a variety of contexts (std, no_std, no_std w/ nostdlib).

I did some research and watched the video linked above. The latest push is my first stab at what I perceive to be "consensus" among the internet without depending on the error-handling crate du jour. However, it will not work in `no_std` due to Boxing the `Error` trait objects.